demo/actions.py

```diff
@@ -378,12 +388,9 @@ def run(self, dispatcher, tracker, domain):
         shown_privacy = tracker.get_slot("shown_privacy")
         name_entity = next(tracker.get_latest_entity_values("name"), None)
         if intent == "greet":
-            if shown_privacy and name_entity and name_entity.lower() != 'sara':
```

Comment:
i don't understand why we've removed all this?

Reply:
I took out the name stuff because this logic was to be able to greet people using `utter_greet_name` and `utter_greet_noname`, which don't exist in the domain file nor in the responses on the platform

Reply:
although i recognize now that this is only here for re-greeting people outside the flow, i can change it so that it's 
```
if shown_privacy and name_entity and name_entity.lower() != 'sara':
    dispatcher.utter_template("utter_greet", tracker)
```
and remove the ask_goal i added to that

Reply:
yes it does exist on the platform, so please put that back :P

Reply:
ack, so sorry. I promise I did actually check -- was a mixture of the not being able to search by utterance name and the platform bug where not all utterance names show up in the filter that made me think it wasn't there 😬noted, will revert. 

side note: i definitely support your desire to be able to download and upload responses from the platform because the out-of-date domain file is very confusing

Reply:
haha yes that it is :D

Reply:
ps, you can totally search by template name, if you use `template:utter_greet`

Reply:
hidden feature haha

Reply:
🤯